File: chapter02/gotoharmfulexample.php
<?php
for ($i=0;$i<1000;$i++){
backtogame:
	$j=0;
	while ($j<1000){
		echo "i = $i and j = $j\n";
		$j++;
	}
}
goto backtogame;
echo "That's all folks!\n";

File: chapter02/gotoharmfulexample.c
#include <stdio.h>

void main() {
	int i,j;

	for (i=0;i<1000;i++){
backtogame:
		j=0;
		while (j<1000){
			printf("i = %d and j = %d\n",i,j);
			j++;
		}
	}
	goto backtogame;
	printf("That's all folks!\n");	
}
